src/pages/questionAnswerViewer/QuestionAnswerViewer.jsx

```diff
@@ -136,6 +155,13 @@ export default function QuestionAnswerViewer() {
                     )) }
                   </Select>
                 </FormControl>
+                {user && user.id_token && (
```

Comment:
This button is currently only rendered when there are existing answers. Might be worth it to move it up so that is displayed even if there are no answers, so that a user can manually trigger an answer lookup if something went wrong when submitting the first time.